### DISCUSSION TITLE
Seamless expressive Fine tuning

### DISCUSSION THREAD

**Discussion by samir1224** [facebookresearch/seamless_communication#330] [Q&A]
Is there any way to use my own dataset for training with the seamless expressive fine tuning models to translate speech to a dialect language ？

**Comment by RRThivyan** [comment]
Hi, one small query. Where can we find the target languages list so that we can finetune our model? Seamless is not allowing all languages to be finetuned. Only a set of languages, if not in list it throws an error.
